lib/sidekiq/ultimate/configuration.rb

```diff
@@ -16,7 +16,36 @@ class Configuration
       #  Sidekiq::Ultimate::Configuration.instance.on_resurrection = ->(queue_name, jobs_count) do
       #    puts "Resurrected #{jobs_count} jobs from #{queue_name}"
       #  end
-      attr_accessor :on_resurrection, :enable_resurrection_counter
+      attr_accessor :on_resurrection
+
+      # resurrection counter can be enabled to count how many times a job was resurrected.
+      # If `enable_resurrection_counter` setting is enabled, on each resurrection event, a counter is increased.
+      # Counter value is stored in redis by jid and has expiration time 24 hours.
+      # @return [Boolean]
+      attr_accessor :enable_resurrection_counter
+
+      # It specifies how often the list of empty queues should be refreshed.
+      # In a nutshell, it specifies the maximum possible delay between a job was pushed to previously empty queue and
+      # the moment when that new job is picked up.
+      # Note that every worker needs to maintain its own local list of empty queues. Setting this interval to a low
+      # values will increase the number of redis calls and will increase the load on redis.
+      # @return [Integer] interval in seconds to refresh the list of empty queues
```

Comment:
Defer some of this explanation to the readme and link it.

```suggestion
      # Each individual worker ignores attempting to fetch jobs from queues it believes are empty by 
      # checking the empty queue cached state from the global empty queue cache to speed up performance
      # @return [Integer] interval in seconds between global cache refreshes
```

Reply:
I'd rather keep it here and link README to these comments. IMO it's easier to keep it up to date than README.

Reply:
I'd also be fine with that to tighten up the Readme language